.github/workflows/ci_tests.yaml

```
@@ -134,9 +134,15 @@ jobs:
           pip install dist/*
 
       # Run the tests
-      - name: Test with pytest
+      - name: Run tests
+        if: github.event_name != 'schedule'
```

This `if` statement means the tests won't be run in daily jobs, which is not what we want. However, removing this `if` statement means on Wednesday job, most tests will be run twice by both `make test` and `make fulltest`.

---

What about something like
```suggestion
        if: github.event.schedule != '0 0 * * 3'
```
Though not sure if it will error for regular (non-scheduled) tests, e.g. in PRs.

---

No errors in PRs.